src/main/java/frc/robot/RobotContainer.java

```diff
@@ -5,59 +5,66 @@
 package frc.robot;
 
 import frc.robot.Constants.OperatorConstants;
-import frc.robot.commands.Autos;
-import frc.robot.commands.ExampleCommand;
-import frc.robot.subsystems.ExampleSubsystem;
+import frc.robot.commands.SwerveCommand;
+import frc.robot.subsystems.Swerve;
+
+import com.pathplanner.lib.auto.AutoBuilder;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.XboxController;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.smartdashboard.SendableChooser;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.button.CommandXboxController;
+import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 import edu.wpi.first.wpilibj2.command.button.Trigger;
 
-/**
- * This class is where the bulk of the robot should be declared. Since Command-based is a
- * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
- * periodic methods (other than the scheduler calls). Instead, the structure of the robot (including
- * subsystems, commands, and trigger mappings) should be declared here.
- */
+
 public class RobotContainer {
-  // The robot's subsystems and commands are defined here...
-  private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
+  
+private Swerve swerve;
+public static boolean centric = true;
+
+public static SendableChooser<Command> autoChooser;
 
-  // Replace with CommandPS4Controller or CommandJoystick if needed
-  private final CommandXboxController m_driverController =
+  private final CommandXboxController driverController =
       new CommandXboxController(OperatorConstants.kDriverControllerPort);
 
-  /** The container for the robot. Contains subsystems, OI devices, and commands. */
+  // private final Joystick driver = new Joystick(OperatorConstants.kDriverControllerPort);
+  // private final JoystickButton robotCentric = new JoystickButton(driver, XboxController.Button.kLeftBumper.value);
+
+
+
+
   public RobotContainer() {
-    // Configure the trigger bindings
+    swerve = new Swerve();
+    swerve.setDefaultCommand(
+      new SwerveCommand(
+        swerve, 
+        () -> driverController.getLeftY(), 
+        () -> driverController.getLeftX(), 
+        () -> driverController.getRightX(), 
+        () -> centric));
+
+    autoChooser = AutoBuilder.buildAutoChooser();
```

Comment:
Do you have any control over what commands this returns?

Reply:
Yes, buildAutoChooser() simply returns a sendable chooser with all the PathPlannerAutos linked to this project. I've only created a simple straight path auto thus far to test.